website/docs/deploy/migration-guide.md

```diff
@@ -42,7 +42,7 @@ const user = userService.loginUserWithoutPassword(
 req.session.user = user;
 ```
 
-- [Read more about Securing Unleash v4](./securing_unleash)
+- [Read more about Securing Unleash v4](./securing-unleash.md)
```

Comment:
It works 😄 It's actually the preferred method according to [the docusaurus docs on linking](https://docusaurus.io/docs/markdown-features/links). That said, I'm not going to update all the old links (there's too many), but instead change them slowly going forward.

Reply:
Sure; feel free to give it a go, but be careful: not all of them work the same way: some of them are relative urls, while others are links to markdown files. Additionally, there's some places where we use an underscore in the file name, but a hyphen in the path name 😬  So if you make it work: great! Just be aware that it might be harder than it looks 🙈